manifests/extension.pp

```diff
@@ -179,24 +179,30 @@
     require => $package_depends,
   }
 
-  # Ubuntu/Debian systems use the mods-available folder. We need to enable
-  # settings files ourselves with php5enmod command.
+  # Ubuntu/Debian systems use the mods-available folder. We need to handle
+  # settings files ourselves with CLI commands.
   $ext_tool_enable   = pick_default($::php::ext_tool_enable, $::php::params::ext_tool_enable)
+  $ext_tool_disable  = pick_default($::php::ext_tool_disable, $::php::params::ext_tool_disable)
   $ext_tool_query    = pick_default($::php::ext_tool_query, $::php::params::ext_tool_query)
   $ext_tool_enabled  = pick_default($::php::ext_tool_enabled, $::php::params::ext_tool_enabled)
+  $final_sapi = $sapi ? {
+    'ALL'   => 'cli',
+    default => $sapi,
+  }
 
   if $::osfamily == 'Debian' and $ext_tool_enabled {
-    $cmd = "${ext_tool_enable} -s ${sapi} ${lowercase_title}"
+    if $ensure == 'absent' {
+      $cmd = "${ext_tool_disable} -s ${final_sapi} ${lowercase_title}"
```

Comment:
I think we should use `${sapi}` instead of `${final_sapi}` here (otherwise the 'ALL' will never work)